packages/@o3r/core/schematics/ng-add/index.ts

```diff
@@ -68,8 +74,9 @@ function ngAddFn(options: NgAddSchematicsSchema): Rule {
           externalPresetRunner?.rule || noop()
         ])(t, c);
       },
-      displayModuleListRule({ packageName: options.projectName })
-    ])(tree, context);
+      options.projectName ? displayModuleListRule({ packageName: options.projectName }) : noop(),
+      setupDependencies(dependenciesSetupConfig)
```

Comment:
should we retrofit some of these fixes to v9 ?

Reply:
Followed by #1354